tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaListenerNameTest.java

```diff
@@ -63,7 +63,7 @@ protected void cleanup() throws Exception {
         // Clean up in the test method
     }
 
-    @Test(timeOut = 30000)
+    @Test
```

Comment:
Why did you remove the timeout?

Reply:
it was only to use the debugger :-) 

restored. good catch